server/src/test/java/com/google/speech/tools/voxetta/BlobstoreLinkServletTest.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
Also add a test for `UtteranceUploadServlet`

Reply:
added